Makefile

```
@@ -44,8 +46,8 @@ setup:
 # Test pipeline with default values
 .PHONY: test
 test:
-	dagger call benchmark-pipeline-test \
-		--source=. --kubeconfig=/src/$(KUBECONFIG)
+	@export PROMETHEUS_URL="http://monitoring-kube-prometheus-prometheus.monitoring:9090" && \
+	dagger call benchmark-pipeline-test --source=. --kubeconfig=/src/$(KUBECONFIG) --prometheus_url=$$PROMETHEUS_URL export --path=$(BENCHMARK_OUTPUT)
```

```suggestion
	dagger call benchmark-pipeline-test --source=. --kubeconfig=/src/$(KUBECONFIG)
```

I think we should make this the default value for the prometheus URL in BenchmarkPipelineTest.

This will fix CI. Otherwise we also need to set it in the workflow
https://github.com/cncf-tags/green-reviews-tooling/blob/main/.github/workflows/benchmark-pipeline-test.yml

---

okay

---

Added the chnges